packages/design-system-tokens/src/tokens/Theme.core.json

```diff
@@ -1149,6 +1149,16 @@
               "codeSyntax": {}
             }
           }
+        },
+        "semibold": {
```

Comment:
just a heads up that this could cause issues. semibold has no meaning on the web. 

1. some fonts don't have a semibold/600 value
2. as far as markup is concerned, there's no "semibold" tag to use in CMS (content management systems) - we only have normal text tags (like `<p>`) and `<strong>`

I wonder if it makes sense to abstract the idea of bold vs not-bold for future theming? Will themes have a need for nuanced weights (probably not since non-retina screens won't be able to differentiate), so maybe having a token representing font weights as "normal" or "default" and "heavy" (or whatever standin word for your bolded type) could prevent the issues presented above?

What I mean is all themes would have a "normal" and "heavy" font weight token, but the value of those tokens could differ across themes. Mgov could have its "heavy" as 700 and CMS  could have its heavy as 600 without the need to give all themes all weights.

Reply:
Could this introduce other issues in a case where a theme wants to use two "heavy" weights (like both 600 and 700)?

Reply:
sure thing!

so first, we used to support bold, semibold and regular font weights (i think also thin at one point?) but we reduced them to the current selection to shrink our bundle size. 

the variable "semibold" was [recently deprecated](https://github.com/CMSgov/design-system/blob/main/packages/design-system/scripts/cmsds-migrate/configs/deprecated-css-v11.mjs#L5) from the DS (as of 3 weeks ago). but the value for it (600 weight) is still active in the system. you can see this on the mgov tokens

variable fonts could solve this problem, but are also large file sizes, so there's a tradeoff as far as asset downloads go. the same asset size issue exists when adding additional font weight files.

Reply:
Great context, thank you! (Also lol for deprecating semibold and then introducing it again now).